nipoppy/trackers/mriqc_tracker.py

```diff
@@ -43,15 +43,14 @@ def check_status(subject_dir, session_id, run_id, file_dict):
 
     return status_msg
         
-def check_T1w(subject_dir, session_id, run_id):
-    return check_status(subject_dir, session_id, run_id, file_dict=T1w_files_dict)
+def check_T1w(subject_dir, session_id, run_id, acq_label=None):
+    return check_status(subject_dir, session_id, run_id, T1w_files_dict)
```

Comment:
Why are we not passing `acq_label` here?

Reply:
Ahh true - it was not needed for me since in my case we are ignoring all the `NM` acq in MRIQC. But for you the default `T1w` have `acq` tag.